manifests/config/exporter/patch/patch-estimator-sidecar.yaml

```diff
@@ -4,10 +4,9 @@ metadata:
   name: kepler-cfm
   namespace: system
 data:
-  CGROUP_METRICS: '*'
   MODEL_CONFIG: |
-    CONTAINER_COMPONENTS_ESTIMATOR=true
-    CONTAINER_COMPONENTS_INIT_URL=https://raw.githubusercontent.com/sustainable-computing-io/kepler-model-server/test_models/tests/test_models/DynComponentPower/CgroupOnly/ScikitMixed.zip
+    NODE_COMPONENTS_ESTIMATOR=true
+    NODE_COMPONENTS_INIT_URL=https://raw.githubusercontent.com/sunya-ch/kepler-model-db/main/models/Linux-4.15.0-213-generic-x86_64_v0.6/rapl/AbsPower/KubeletOnly/GradientBoostingRegressorTrainer_1.zip
```

Comment:
is the `models` directory also available at `https://github.com/sustainable-computing-io/kepler-model-db/`?

Reply:
Not yet. I'm planning to have a PR template before but do you think should I merge it first?

Reply:
Thanks. Fix the repo.